migrations/postgres/20240422100311_flow_configuration_events.sql

```diff
@@ -0,0 +1,26 @@
+CREATE SEQUENCE flow_configuration_event_id_seq AS BIGINT;
+
+CREATE TYPE system_flow_type AS ENUM ('gc');
+
+CREATE TABLE system_flow_configuration_events
```

Comment:
Might be a little easier with just 1 table, but that is not critical.

Reply:
In general, as far as I understand the future usage, we will have 99% dataset of events, and for the sake of 1% of system events, it is not very desirable to make optional columns

Reply:
We don't know yet if system flows will have any configuration at all. But I can imagine a scheduled GC, once a week i.e.